compiler/src/iree/compiler/Dialect/Stream/Conversion/FlowToStream/Patterns.cpp

```diff
@@ -126,7 +114,7 @@ struct ConvertTensorCloneOp
     rewriter.replaceOpWithNewOp<IREE::Stream::TensorCloneOp>(
         op, unknownType, operand.resource, op.getOperand().getType(),
         op.getArgumentDims(), operand.resourceSize, op.getResult().getType(),
-        adaptor.getArgumentDims(), operand.resourceSize, getAffinityFor(op));
+        adaptor.getArgumentDims(), operand.resourceSize, IREE::Stream::AffinityAttr::lookup(op));
```

Comment:
clang-format

https://github.com/openxla/iree/actions/runs/7214291070/job/19656051828?pr=15929

Reply:
weird - go home vscode, you're drunk